### ISSUE TITLE
Missing content in Version 3.0.1

### ISSUE THREAD

**Issue by jonaswjs** [LaxarJS/ax-developer-tools-widget#70]
In version 3.0.1 the content directory is missing in dist version. When trying to install the ShopDemo this error message appears:
```
Module not found: Error: Can't resolve './content/var/dist/' in '/export/source/jschulte/test-shop/shop-demo/node_modules/laxar-developer-tools-widget'
```

**Comment by x1B** [comment]
@jonaswjs thx for the report. I'll look into it

**Comment by x1B** [comment]
The bundle for v3.0.1 was broken because I hadn't pulled in the `content` submodule. The v3.0.2 package should work as expected. For an existing shop-demo installation, it should be enough to remove+reinstall the laxar-developer-tools-widget from `node_modules`, or to manually `npm install laxar-developer-tools-widget@3.0.2`.

**Comment by x1B** [comment]
Fixed on `master` (`v3.0.2`).
